db/controllers/news.controllers.js

```diff
@@ -0,0 +1,9 @@
+const{selectTopics} = require('../models/news.models.js')
+
+exports.getTopics = (request, response) => {
+    selectTopics()
+.then((topics) => {
+response.status(200).send({topics})
+
+})
```

Comment:
do you think a catch block would be helpful here 💭

Reply:
yup, I am confused about catch blocks and errors and whether or not to have a separate file at this stage for errors, but will look at the docs and try to sort for next one, thank you